data/apple

```diff
@@ -800,6 +808,7 @@ full:app-site-association.cdn-apple.com @cn
 full:appldnld.apple.com @cn
 full:appldnld.g.aaplimg.com @cn
 full:appleid.cdn-apple.com @cn
+full:apps.apple.com @cn
```

Comment:
Could you provide any evidence of it?

Reply:
Unless #503 is no longer the case, this particular domain should not be included.

Maybe keep it as a comment to remind future maintainers?

Reply:
I do agree.